applications/luci-app-tor-hs/root/usr/share/rpcd/acl.d/luci-app-tor-hs.json

```diff
@@ -0,0 +1,16 @@
+{
+	"luci-app-tor-hs": {
+		"description": "Grant access to Tor hidden service configurator",
+		"read": {
+			"ubus": {
+				"file":["read","list"],
+				"tor-hs-rpc": ["list-hs"]
+			},
+			"uci": [  "tor-hs" ],
+			"file": {"/etc/tor":["list","read"]}
+		},
+		"write": {
+			"uci": ["tor-hs"]
+		}
+	}
+}
```

Comment:
remove one `<space>` before "tor-hs" in read.
Also add `<spaces>` after and before each bracket.